crates/sui/src/client_commands.rs

```diff
@@ -1658,35 +1661,123 @@ fn convert_number_to_string(value: Value) -> Value {
 }
 
 // TODO(chris): only print out the full response when `--verbose` is provided
-pub fn write_transaction_response(
+// pub fn write_transaction_response(
+//     response: &SuiTransactionBlockResponse,
+// ) -> Result<String, fmt::Error> {
+//     let mut writer = String::new();
+//     writeln!(writer, "{}", "----- Transaction Digest ----".bold())?;
+//     writeln!(writer, "{}", response.digest)?;
+//     writeln!(writer, "{}", "----- Transaction Data ----".bold())?;
+//     if let Some(t) = response.transaction {
+//         writeln!(writer, "{}", t)?;
+//     }
+
+//     writeln!(writer, "{}", "----- Transaction Effects ----".bold())?;
+//     if let Some(e) = response.effects {
+//         writeln!(writer, "{}", e)?;
+//     }
+
+//     writeln!(writer, "{}", "----- Events ----".bold())?;
+//     if let Some(e) = response.events {
+//         writeln!(writer, "{:#?}", json!(e))?;
+//     }
+
+//     writeln!(writer, "{}", "----- Object changes ----".bold())?;
+//     if let Some(e) = response.object_changes {
+//         writeln!(writer, "{:#?}", json!(e))?;
+//     }
+
+//     writeln!(writer, "{}", "----- Balance changes ----".bold())?;
+//     if let Some(e) = response.balance_changes {
+//         writeln!(writer, "{:#?}", json!(e))?;
+//     }
+//     Ok(writer)
+// }
```

Comment:
Delete?

Reply:
Weird, I removed this already. Might have not committed properly or something :|